org.mobsya.ThymioSuite.json

```
@@ -0,0 +1,117 @@
+                "--disable-gdbm",
+                "--disable-libdaemon",
+                "--disable-libevent",
+                "--disable-manpages",
+                "--with-systemdsystemunitdir=/app/lib/systemd"
+            ],
+            "cleanup": [
+                "/bin", "/include", "*.a", "/lib/pkgconfig", "/lib/avahi",  "/lib/girepository-1.0"
+            ]
+        },
+        {
+            "name": "boost",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url":  "http://dl.bintray.com/boostorg/release/1.70.0/source/boost_1_70_0.tar.gz",
+                    "sha256": "882b48708d211a5f48e60b0124cf5863c1534cd544ecd0664bb534a4b5d506e9"
+                }
+            ],
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install --prefix=/app/ variant=release link=static threading=multi --with-filesystem --with-atomic --with-thread --with-chrono --with-system --with-date_time --with-regex --with-serialization --with-program_options -j 4"
+            ],
+            "cleanup": [
+                "/lib/", "/lib/debug", "/include", ""
+            ]
+        },
+        {
+            "name": "ThymioSuite",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DFLAT_PACK_BUILD=ON",
+                "-DBOOST_ROOT=/app",
+                "-DBOOST_LIBRARYDIR=/app/lib",
+                "-DBOOST_INCLUDEDIR=/app/include",
+                "-DBoost_DETAILED_FAILURE_MSG=ON"
+            ],
+            "sources": [
+                {
+                    "type": "git",
+                    "url" : "https://github.com/Mobsya/aseba",
+                    "commit" : "a7876803a0f89315fe2cac78665fe708a9f8e582"
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Mobsya/thymio-blockly-standalone/releases/download/v20190507.3/thymio-blockly.tar.gz",
+                    "sha256": "0a106c7a52a9e8998abcbec203e26ece45562e14b106ee17fa52375b75dc1be3",
+                    "strip-components": 0
+                },
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Mobsya/scratch-gui/releases/download/v20190606.2/scratch-gui.tar.gz",
+                    "sha256": "660ae22633f839fea83a5f8e8d7d5d93c9c9998782c058f855e430c8a62bff8a",
+                    "strip-components": 0
+                },
+                {
+                    "type": "file",
+                    "path": "org.mobsya.ThymioSuite.appdata.xml"
+                }
+            ],
+            "post-install": [
+                "install -D --target-directory /app/share/mobsya/Playground/  aseba/targets/playground/examples/*",
+                "install -D --target-directory /app/share/appdata/ org.mobsya.ThymioSuite.appdata.xml"
```

```suggestion
                "install -D --target-directory /app/share/metainfo/ org.mobsya.ThymioSuite.metainfo.xml"
```
Please rename the appdata file to metainfo.